src/main/java/com/example/shorturl/controller/api/ShortenUrlControllerV2.java

```diff
@@ -33,11 +36,15 @@ public ResponseEntity<UserUrlResponse> shortUrl(@RequestBody UserUrlRequest user
         if(Objects.isNull(userUrlRequest) || StringUtils.isEmpty(userUrlRequest.getUrl()))
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Url request is missing or empty");
 
-        UrlEntity url = shortenUrlService.userShortUrl(userUrlRequest);
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        MyUserDetails userEntity = (MyUserDetails) authentication.getPrincipal();
+        Long userId = userEntity.getId();
+
+        UrlEntity url = shortenUrlService.userShortUrl(userId, userUrlRequest);
 
         return new ResponseEntity<>(
                 UserUrlResponse.builder()
-                        .shorUrl(url.getShortUrl())
+                        .shortUrl(url.getShortUrl())
```

Comment:
I am thinking, maybe we should also add `userId` to response? What do you think?

Reply:
Sure, It gives more information, let me add it

Reply:
Added